### ISSUE TITLE
Process complete MERFISH U2-OS dataset and verify results are biologically reasonable

### ISSUE THREAD

**Issue by ambrosejcarr** [spacetx/starfish#850]
This issue contains multiple steps: 

- [x] image data is located `s3://spacetx.starfish.data/moffit_u20s/dataByFOV/`
- [x] codebook is located `s3://spacetx.starfish.data/moffit_u20s/L26E1_codebook.csv` but can probably be taken from existing MERFISH test data.
- [x] coordinates are located `s3://spacetx.starfish.data/moffit_u20s/stagePos.csv`
- [x] scale factors must be recorded in the `experiment.json`
- [x] data is formatted & validated
- [x] write pipeline
- [ ] publish `DecodedSpots`

**Comment by ambrosejcarr** [comment]
@dganguli `stagePos.csv` is just a series of unlabeled pairs. Can you guess the organization? Which column is `x` and which is `y`, and are rows sequentially ordered from fov 0? The number of rows appears to match (496). 

```
$ head stagePos.csv 
-1500,-3000
-1500,-2800
-1500,-2600
-1500,-2400
-1500,-2200
-1500,-2000
-1500,-1800
-1500,-1600
-1500,-1400
-1500,-1200
```

**Comment by dganguli** [comment]
I'm not sure, based on the data. But my prior is that it's ordered from FOV 0 to 496, and the columns are y, x. Another option is to reach out to the spacejammer from this group to ask? @neuromusic

**Comment by ambrosejcarr** [comment]
Data is uploaded to `s3://spacetx.starfish.data.public/browse/formatted/MERFISH/20190511/`

**Comment by ambrosejcarr** [comment]
@shanaxel42 we decided we'd forego analysis of biological results for the time being because we don't have a good handle on segmentation yet. This means the last step is to publish the `DecodedSpots` somewhere. I don't know where those are so I've assigned this over to you.

**Comment by shanaxel42** [comment]
results uploaded here! https://console.aws.amazon.com/s3/buckets/starfish.data.published/MERFISH/20181005/starfish_results/?region=us-east-1&tab=overview
